servers/rendering/renderer_rd/shaders/forward_mobile/scene_forward_mobile.glsl

```diff
@@ -1489,7 +1515,7 @@ void main() {
 #if !defined(MODE_RENDER_DEPTH) && !defined(MODE_UNSHADED)
 #ifdef USE_VERTEX_LIGHTING
 	diffuse_light += diffuse_light_interp.rgb;
-	specular_light += specular_light_interp.rgb * f0;
+	indirect_specular_light += specular_light_interp.rgb * f0;
```

Comment:
This and the next couple should be `direct_specular_light` since vertex lighting is all direct lighting